examples/boxed_values/output.txt

```diff
@@ -1,3 +1,6 @@
+user is pointer: *models.User
```

Comment:
This would read a bit better if it said "type of user result" or something along those lines. Not really a big deal though.

Reply:
Yea that is a good idea, I will change it